partiql-eval/src/lib.rs

```diff
@@ -464,22 +480,506 @@ mod tests {
         }
     }
 
+    // Spec 6.1:
+    //  SELECT VALUE 2*v.a
+    //  FROM [{'a': 1}, {'a': 2}, {'a': 3}] AS v;
+    //  Expected: <<2, 4, 6>>
     #[test]
     fn select_value() {
-        // Plan for `select value a from data`
-        // TODO
+        // Plan for `select value a as b from data`
+        let mut lg = LogicalPlan::new();
+
+        let from = lg.add_operator(scan("data", "v"));
+
+        let va = path_var("v", "a");
+        let select_value = lg.add_operator(ProjectValue(logical::ProjectValue {
+            expr: ValueExpr::BinaryExpr(
+                BinaryOp::Mul,
+                Box::new(va),
+                Box::new(ValueExpr::Lit(Box::new(Value::Integer(2)))),
+            ),
+        }));
+
+        let sink = lg.add_operator(BindingsExpr::Sink);
+
+        lg.add_flow(from, select_value);
+        lg.add_flow(select_value, sink);
+
+        if let Value::Bag(out) = evaluate(lg, data_3_tuple()) {
```

Comment:
It may be worth bringing in the [assert_matches crate](https://docs.rs/assert_matches/latest/assert_matches/macro.assert_matches.html) to allow something like:

```rust

        let output = evaluate(lg, data_3_tuple());
        assert_matches!(output, Value::Bag(bag) => {
            assert_eq!(*bag, partiql_bag![2, 4, 6]);
        });

        ```

Reply:
I'll add it to the next commit in this PR.